src/datasources/balances-api/coingecko-api.service.ts

```diff
@@ -241,7 +241,8 @@ export class CoingeckoApi implements IPricesApi {
       const { key, field } = cacheDir;
       if (cached != null) {
         this.loggingService.debug({ type: 'cache_hit', key, field });
-        result.push(JSON.parse(cached));
+        const cachedAssetPrice: AssetPrice = JSON.parse(cached);
+        result.push(cachedAssetPrice);
```

Comment:
Nit (as it was technically like this before): `JSON.parse` returns `unknown` and we're asserting it - I think a type guard would be cleaner but we can address it later.

I won't comment on this elsewhere as I think it's fine to leave as is for not but wanted to keep record of it.

Reply:
I agree, I've left a TODO to add a validator/type guard for this entity, even if it's Coingecko-specific, it would give us type-safety.

Reply:
[5118bed](https://github.com/safe-global/safe-client-gateway/pull/1533/commits/5118beddfa2bdb98fd00114e21ae415ecfbc903e)